### ISSUE TITLE
Registration Status of Teams

### ISSUE THREAD

**Issue by bguerrettaz** [YBNetwork/Issues-Enhancements#4]
Right now, we have a lot of teams that register for a tournament, but do NOT pay at the time they register.  Therefore, we need to create a payment portal for them on their coaches dashboard for a specific tournament.  Most of this is already created on the coaches dashboard, under VIEW TOURNAMENT Status. Below you will find the picture.  What I would like to do
 
1. Create another column that say's PAY NOW after Registration status.
2. When user clicks on PAY NOW, basically the payment process comes just like a team sees when they registered for the tournament the first time.
3. Just need to make sure it doesn't put a "double registration" into the Directors Dashboard - Registration Report.  It should just allow the team to pay and change the team's status to PAID if using credit card, or allow the coach (user) to see where to send the payment and the amount of the payment.

**Comment by bguerrettaz** [comment]
Is this still the top priority?

**Comment by kirkwhiteman1** [comment]
Yes, this is still a top priority.  I actually added some additional information in Issue #8 that is on the same page as this issue

**Comment by bguerrettaz** [comment]
Ok, so 8 and 4 should be done at the same time?

**Comment by kirkwhiteman1** [comment]
Yes.

 

Yours in Hoops,

 

Kirk Whiteman

217.714.7263

 <http://www.aybtour.com/> 

 <http://www.ybnetwork.net/>      <https://www.facebook.com/americanyouthbasketballtour/>      <https://twitter.com/AybTour>      <https://www.instagram.com/aybtour/>

**Comment by bguerrettaz** [comment]
Estimate for 4: 6-10 hours

I will estimate #8 in that issue.

**Comment by bguerrettaz** [comment]
@kirkwhiteman1 - I have completed the initial changes for this issue.  The column was added for 'Pay Now' with the button reading 'Pay'.  Once they click 'Pay', they are taken to the Pay/Register site.  They will work through the existing workflow, but the difference is, if they select a team they have already registered, but not paid for, it will not register that team again, but it will allow them to pay.

I also made it capable of selecting multiple teams, some can be registered and others not, and it will register those that need to be registered and not duplicate the others registration and take you to pay for all selected.

I have not completed #8 , once you confirm this issue looks good, I'll move directly into #8.  Please let me know any changes you would like to see (verbiage, workflow, etc.) or if you find any issues.

Thanks,
Blake

**Comment by kirkwhiteman1** [comment]
So far after my review, it looks good.  I have not gone through the process of actually paying through PayPal yet, but some teams will be doing this today.  We are ready to move on #8

**Comment by bguerrettaz** [comment]
Sounds good, I'll work on #8 now.

**Comment by kirkwhiteman1** [comment]
Blake: Upon further review of this issue.  As I was browsing our "Tournament listings" from the main page and not logged in to either the coach's dashboard or director's dashboard, I noticed that when I click the "register" button to view more tournament details, the Button at the top still says "Click here to PAY".

I was under the impression that the "Click here to PAY" would only show when this page was accessed through the new payment portal created from the Coaches Dashboard - My Teams - View Tournament Status; for teams that had already registered, but not paid.

Is there anyway this button can be variable?  Meaning:
1. When a coach (team) accesses this page from the main page of the website, the button would read "Register Now"
2. When a coach (team) accesses this page from their "Coach's Dashboard-My Teams - View Tournament Status - Pay Now button - the button will read "Click here to PAY"

My concern, many teams want to Register, but not Pay right away.  My concern is that if a coach is looking at the tournament for the first time from the main page and sees "Click here to PAY", the coach may not even register for the tournament.

Let me know if you understand and if this is possible?
